lib/omnibus/exceptions.rb

```diff
@@ -317,4 +317,10 @@ def initialize(version)
 EOF
     end
   end
+
+  class FailedToTimestamp < Error
```

Comment:
This seems like a pretty generic error, but it only appears to be used in the Windows timestamping. I think we do other kinds of timestampping. Would that error still apply and could be fixed in another PR? If this is a different kind of timestampping failure, maybe we could make the error more specific?

Also, we usually put the error text in the error itself (see above for examples). It looks like this is putting the error on the responsibility of the caller.